src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResults.tsx

```diff
@@ -0,0 +1,11 @@
+import { Screen, Text } from "palette"
+
+export const ArtQuizResults = () => {
+  return (
+    <Screen>
+      <Screen.Body>
+        <Text>Result screen in progress ..</Text>
```

Comment:
```suggestion
        <Text>Result screen in progress...</Text>
```

i removed the space, but then I saw we use `...` elsewhere and not just `..` that I like, so here it is haha

Reply:
This will be replaced in the next PR anyway. 😉